### PR TITLE
Remove devops from Codeowners

### PR DESCRIPTION
**Pull request by annaswims**
because it's too noisy to tag `@department-of-veterans-affairs/ad-hoc-devops` 

While it's fine to have the devops team review PR's when they see fit, it's not worth the extra noise of them being tagged on every PR, so I'm removing them from the CODEOWNERS file.  Should it be necessary, someone from `@department-of-veterans-affairs/frontend-review-group` can rubber-stamp a PR.

### CODE REVIEW

**Review by tskinn — APPROVED** [review]
Thanks @annaswims !!

**Comment by gunsch** [comment]
I'm still confused why we wouldn't want to use precedence like the example here?https://help.github.com/en/articles/about-code-owners

seems like if instead of:

```
*       @department-of-veterans-affairs/frontend-review-group @department-of-veterans-affairs/ad-hoc-devops
```

you had:

```
*       @department-of-veterans-affairs/ad-hoc-devops 
*       @department-of-veterans-affairs/frontend-review-group
```

then both groups would be owners,  but it would only notify `frontend-review-group`. thoughts?

**Comment by annaswims** [comment]
My interpretation of `# Order is important; the last matching pattern takes the most precedence.` is that in the example you have, `@department-of-veterans-affairs/frontend-review-group` would match all files and therefore be the code owner of every file.  

so if `CODEOWNERS ` was 
````
*.md @gunsch 
*  @department-of-veterans-affairs/frontend-review-group
````
You wouldn't own anything, but if it was:

````
*  @department-of-veterans-affairs/frontend-review-group
*.md @gunsch 
````
then you would own all .md files

**Comment by gunsch** [comment]
Yeah, that's not how I read that at all. I think it pretty clearly calls out that it will notify the last matching person for _review_, but I don't think that changes who has ownership (in case of needing to approve/submit in a pinch).

i.e. in the GitHub example, I don't think "@global-owner1" loses ownership of JS files just because "@js-owner" also has ownership of JS  files --- but just means that "@global-owner1" won't get notified for every single JS code review.

**Comment by annaswims** [comment]
I decided to give it a spin on vets.gov-team, because it's 

It appears if I have a CODEOWNERS of 

````
README.md @gunsch 
*  @rianfowler 
````

and I edit just README.md then `@rianfowler` appears to be the only code owner. 

Screen Shot 2019-07-31 at 4 09 43 PM

**Comment by gunsch** [comment]
Boo, okay. Seems like GitHub really conflates the "automatic review requested" with "who's _required_ to get approval from" in a way I wasn't expecting.

I suppose we could also get to the state you wanted by making the devops group repository admins (if they aren't already) 😂

**Pull request merged**
